gcc/rust/ast/rust-expr.h

```diff
@@ -6,6 +6,8 @@
 #include "rust-path.h"
 #include "rust-macro.h"
 #include "rust-operators.h"
+#include "rust-system.h"
+#include <memory>
```

Comment:
ah, the system include is still here. you should be able to remove it entirely

Reply:
gotcha